compiler/prelude/TysPrim.hs

```diff
@@ -369,6 +370,8 @@ openBetaTy  = mkTyVarTy openBetaTyVar
 multiplicityTyVar :: TyVar
 multiplicityTyVar = mkTemplateTyVars (repeat multiplicityTy) !! 13
 
+multiplicityTyVarList :: [a] -> [TyVar]
+multiplicityTyVarList l = take (length l) $ drop 13 $ mkTemplateTyVars (repeat multiplicityTy)
```

Comment:
Doesn't it make `multiplicityTyVar`, above, dead?

Could it cause name conflicts? (I was wondering the same for `multiplicityTyVar`, I don't know whether it's the right way).

Reply:
`multiplicityTyVar` is still used in `funTyCon`.

I'm not sure whether `multiplicityTyVar` is the right approach, either. However, this PR shouldn't introduce new conflicts.